package.json

```
@@ -6,28 +6,27 @@
     "node": ">=20.0.0"
   },
   "scripts": {
-    "prepare": "gulp bundle --color",
-    "build": "gulp build --color",
+    "build": "gulp bundle docs:build --color --series",
+    "prestart": "npm run build",
     "start": "gulp --color",
-    "test": "npm run lint && npm run jest:ci",
+    "pretest": "npm run build",
+    "test": "jest --color --coverage",
     "lint": "npm run lint:js && npm run lint:css && npm run prettier",
     "prettier": "prettier --cache --cache-location .cache/prettier --cache-strategy content --check .",
     "prettier:fix": "prettier --write .",
     "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore \"**/*.js\"",
     "lint:js:fix": "npm run lint:js -- --fix",
     "lint:css": "stylelint --cache --cache-location .cache/stylelint --cache-strategy content --color --formatter verbose --ignore-path .gitignore \"**/*.{md,scss}\"",
     "lint:css:fix": "npm run lint:css -- --fix",
-    "backstop:interactive": "backstop remote --config ./tests/backstop/backstop.js",
-    "backstop:ref": "concurrently \"npm start\" \"wait-on http://localhost:3000 && backstop --config=./tests/backstop/backstop.js reference --docker\"",
-    "backstop:test": "concurrently \"npm start\" \"wait-on http://localhost:3000 && backstop --config=./tests/backstop/backstop.js test --docker\"",
-    "backstop:ci": "start-server-and-test start http://0.0.0.0:3000 backstop:citest",
-    "backstop:citest": "backstop --config=./tests/backstop/backstop.js test --docker",
-    "backstop:approve": "backstop --config=./tests/backstop/backstop.js approve",
+    "backstop:interactive": "backstop --config backstop.config.js remote",
+    "prebackstop:cli": "npm run build",
+    "backstop:cli": "concurrently \"npm start --ignore-scripts\" \"wait-on http://0.0.0.0:3000 && backstop --config backstop.config.js {@}\" --kill-others --passthrough-arguments --success first",
+    "backstop:ref": "npm run backstop:cli -- -- reference --docker",
+    "backstop:test": "npm run backstop:cli -- -- test --docker",
```

Worth renaming under the `test:` namespace, to make it a bit clearer what this does? (could also be `test:rendering` or `test:visualbackstop` or something else).

```suggestion
    "test:visual": "npm run backstop:cli -- -- test --docker",
```

Also then potentially allows us to have other types of tests in future like `test:html` or `test:javascript` or whatever, with `npm test` running them all (except the visual ones).